src/app/channels/channels.module.ts

```diff
@@ -5,6 +5,7 @@ import { ChannelsRoutingModule } from './channels-routing.module';
 import { ChannelsComponent } from './channels.component';
 import { SharedModule } from '../_shared/shared.module';
 import { ChannelComponent } from './channel/channel.component';
+// import { YoutubeModule } from './youtube/youtube.module';
```

Comment:
I think we can remove this. It is time we do some clean up. 
If it is comment out for years... I don't think anything bad happen in deleting it. 
And anyway, if we need it back, we can always check Github history.

Reply:
I don't think this PR should make code changes that are not strictly related to getting everything working with Angular 16, that's why I reverted the change to this line (and some style changes).

Reply:
Ok, makes sense to keep things as close as possible to the upgrade and reduce confusion.